conf.py

```diff
@@ -176,9 +176,13 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#filtering
 linkcheck_anchors_ignore_for_url = [
     'https://github.com/',
-    'https://index.ros.org/'
+    'https://index.ros.org/',
+    'https://blogs.oracle.com/'
```

Comment:
```suggestion
    'https://index.ros.org/'
```
I think this is not necessary here, as this is only for anchors?